docs/design/features/Windows-Arm64-UX.md

```diff
@@ -0,0 +1,21 @@
+# Windows Arm64 User Experience
+
+## Mixed Architecture Side By Side Installs
+
+This design is incomplete.  A rough sketch follows.
+
+### Proposed Side-By-Side Design
+
+For Windows the proposed plan is to:
+
+- On Windows Arm .NET runtime versions TBD will always require `x64` architecture. All other runtime versions will require `arm64` architecture.
```

Comment:
Note that .NET 5 shipped for Windows ARM64 already.

Reply:
Does this mean we no longer have an opportunity to design the Windows Arm64 user experience.  If so, I can remove this doc.

Reply:
We are starting from a different place on Windows. We have opportunity to evolve the experience. Also, the tools/APIs at our disposal are different between Windows and macOS.